### PR TITLE
fix: connector bugs

### PR DESCRIPTION
**Pull request by seunghwako**
# Test before submitting

Before submitting a Pull Request, please test your code. If you created a new created a new feature, then create the test function, too. Please refer to [this document](../CONTRIBUTING.md).

Put the name of the test function of the service you created in the command below.
This is a regular expression that returns all tests containing the character string, so if there are several test codes, you can put a keyword including all of the test codes and turn them.
If you omit the include statement and operate the test, all the test codes we write will work, and myriad errors will occur in your code without environmental variables.

```ts
npm run build && npm run test -- --include TEST_FUNCTION_NAME
```

Of course you won't be able to provide us with your environmental variables, so we'll replace them with our environmental variables to run the program and merge it depending on whether it passes or not.

# Purpose

Please explain to our developers what the features you modified or developed are for.

# Check List

- [ ] Passing an existing or newly created test
- [ ] sufficient summary and description for LLM to understand
- [ ] Does it ensure backward compatibility with existing features, or does the modifications not affect pre-modification request, response type?

### CODE REVIEW

File: src/controllers/connector/csv/CsvModule.ts [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import { Module } from "@nestjs/common";
 import { CsvController } from "./CsvController";
+import { AwsProvider } from "../../../providers/connector/aws/AwsProvider";
+import { CsvProvider } from "../../../providers/connector/csv/CsvProvider";
 
 @Module({
   controllers: [CsvController],
-  providers: [],
+  providers: [AwsProvider, CsvProvider],
```
Comment (kakasoo) [inline src/controllers/connector/csv/CsvModule.ts:8]: AwsProvider는 AwsModule import 하는 것으로!

File: src/controllers/connector/excel/ExcelModule.ts [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import { Module } from "@nestjs/common";
 
 import { ExcelController } from "./ExcelController";
+import { ExcelProvider } from "../../../providers/connector/excel/ExcelProvider";
+import { AwsProvider } from "../../../providers/connector/aws/AwsProvider";
 
 @Module({
   controllers: [ExcelController],
-  providers: [],
+  providers: [ExcelProvider, AwsProvider],
```
Comment (kakasoo) [inline src/controllers/connector/excel/ExcelModule.ts:9]: AwsModule import하는 것 동일.

**Review by kakasoo — APPROVED** [review]
module import만 해주세용 이거 internal 코드들인데 테스트 로컬에서 돌려보셨을까요?

File: src/controllers/connector/csv/CsvModule.ts (hunk 1 above)
Comment (seunghwako) [reply src/controllers/connector/csv/CsvModule.ts]: done~

File: src/controllers/connector/excel/ExcelModule.ts (hunk 2 above)
Comment (seunghwako) [reply src/controllers/connector/excel/ExcelModule.ts]: done~

**Pull request merged**